lib/FullScreenView/tests/FullScreenView-test.js

```diff
@@ -36,35 +38,53 @@ describe('FullScreenView', () => {
   describe('rendering FullScreenView', () => {
     const paneTitle = 'Pane title';
     const handleCancel = sinon.spy();
+    const renderHeader = sinon.spy();
 
-    beforeEach(async () => {
-      handleCancel.resetHistory();
+    describe('when renderHeader prop is not passed', () => {
+      beforeEach(async () => {
+        handleCancel.resetHistory();
 
-      await setupFullScreenView({
-        paneTitle,
-        handleCancel,
-        children: <span data-test-children>Children</span>,
+        await setupFullScreenView({
+          paneTitle,
+          handleCancel,
+          children: <span data-test-children>Children</span>,
+        });
       });
-    });
 
-    it('should be present', () => {
-      expect(fullScreenView.isPresent).to.be.true;
-      expect(fullScreenView.headerTitle).to.equal(paneTitle);
-      expect(fullScreenView.closeButton.isPresent).to.be.true;
-      expect(fullScreenView.actionMenu.isPresent).to.be.true;
-    });
+      it('should be present', () => {
+        expect(fullScreenView.isPresent).to.be.true;
+        expect(fullScreenView.headerTitle).to.equal(paneTitle);
+        expect(fullScreenView.closeButton.isPresent).to.be.true;
+        expect(fullScreenView.actionMenu.isPresent).to.be.true;
+      });
 
-    it('should contain children', () => {
-      expect(fullScreenView.$('[data-test-children]')).to.not.be.undefined;
+      it('should contain children', () => {
+        expect(fullScreenView.$('[data-test-children]')).to.not.be.undefined;
+      });
+
+      describe('clicking on close button', () => {
+        beforeEach(async () => {
+          await fullScreenView.closeButton.click();
+        });
+
+        it('should call the subscribed callback', () => {
+          expect(handleCancel.called).to.be.true;
+        });
+      });
     });
 
-    describe('clicking on close button', () => {
+    describe('when renderHeader prop is passed', () => {
```

Comment:
Please, use the consistent format for messages that is used in stripes-data-transfer-components.. 

Please, use the consistent format for messages that is used in stripes-data-transfer-components.. 

Some examples for the describe:
describe('clicking on close button', () => {
describe('rendering FullScreenView', () => {

Reply:
changed